CHANGELOG.asciidoc

```diff
@@ -35,6 +35,15 @@ endif::[]
 [[release-notes-4.x]]
 === Ruby Agent version 4.x
 
+[[release-notes-4.6.0]]
+==== 4.6.0
```

Comment:
What's the reason behind this being in a new major and not the next minor?

Reply:
I think the next version is going to include a couple (minor) features, so semver says minor version. Not a big deal either way, but that's my plan at the moment.